comet/core/models.py

```
@@ -404,7 +434,98 @@ def model_post_init(self, __context):
+        raise RuntimeError(error_context) from None
+    except Exception as error:
+        logger.error(f"Failed to persist {token_name}_FILE ({token_file}): {error}")
+        raise
+
+
+def _build_stremio_api_prefix():
+    should_protect_api = bool(settings.CONFIGURE_PAGE_PASSWORD) or bool(
+        settings.PUBLIC_API_TOKEN
+    )
+    if not should_protect_api:
+        settings.PUBLIC_API_TOKEN_SOURCE = "disabled"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "PUBLIC_API_TOKEN_SOURCE"

(S105)